README.md

```diff
@@ -1,131 +1,130 @@
 # The Relying Party Transmitter
 
-This is the source code for the Relying Party Transmitter (RPT) which is responsible for transmitting messages from Relying parties (RPs) to the Shared Signals Framework (SSF).
+The Relying Party Transmitter (RPT) application demonstrates how a [Security Event Token](https://datatracker.ietf.org/doc/html/rfc8417) (SET) can be sent from Relying Parties (RPs) to the Transmitter in the Shared Signals Framework (SSF) using a severless function, known as the Transmitter function. This function signs and encrypts a Security Event Token and forwards it to the Shared Signals Framework endpoint.
+
+Helper functions have been added for the purpose of testing the Transmitter function. The Generator function generates test cases of messages to pass on and the public key function simulates an RP's endpoint for serving their public key.
+
+Though this implementation uses serverless functions in AWS (AWS Lambda), the approach described is platform independent.
+
 ![Architecture of the RPT](/READMEresources/MainArchitecture.png)
 
----
+## Background
 
-# 1. Setup
+A Security Event Token is issued on a state change of a security subject, for example a user account or an HTTP session. When the Shared Signals Transmitter endpoint receives a SET, it will validate and interpret the received Security Event Token and takes its own independent actions, if any.
 
----
+An example Security Event Token:
 
-## 1.1. Development Environment Set Up
+```json
+{
+   "SET": {
+      "iss": "https://issuer.digitalIdentity.gov/",
+      "jti": "756e697175654964656e74696669657230",
+      "iat": 1520364019,
+      "aud": "https://audience.hmrc.gov/",
+      "events": {
+      "https://schemas.digitalIdentity.gov/secevent/risc/event-type/ipv-spot-request-recieved": {
+         "subject": {
+            "format": "iss_sub",
```

Comment:
The iss-sub user subject format has been deprecated in favour of { "format" : "uri", "uri" : "<some-user-uri>" }. Not entirely sure this is a user subject though? I might be inclined to use a different message type in the example here, maybe one that clearly has a user subject? Maybe RISC account disabled?